site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -75,9 +76,7 @@ const TerminalPage: FC<
       fontFamily: MONOSPACE_FONT_FAMILY,
       fontSize: 16,
       theme: {
-        // This is a slight off-black.
-        // It's really easy on the eyes!
-        background: "#1F1F1F",
+        background: colors.gray[16],
```

Comment:
This is fine but `palette.background.default` is `gray[15]` so it might work as well. If the darker one looks better though, no problem.